_includes/configuration/toolbar-mode.md

```diff
@@ -6,6 +6,25 @@
 
 The `toolbar_mode` option is used to extend the toolbar to accommodate the overflowing toolbar buttons. This option is useful for small screens or small editor frames.
 
+The editor will move toolbar buttons to the toolbar drawer based on:
+
+- The defined toolbar groups.
+- The width of the editor.
+
+For example. If the following toolbar configuration is wider than the editor, the toolbar will be split at the group separator (`|`).
+
+```js
+tinymce.init({
+    selector: 'textarea',
+    toolbar: 'undo redo styleselect bold italic alignleft aligncenter alignright alignjustify | bullist numlist outdent indent'
+});
+```
+
+Two situations to consider when organising the toolbar:
+
+- If the first toolbar button group is wider than the editor, the whole toolbar will be shown in the toolbar drawer.
+- If no toolbar button groups are defined, the `toolbar_mode` will default to [the `wrap` toolbar mode](#wrap).
```

Comment:
Is this true? If it is, it's definitely a bug as the whole toolbar should appear in the drawer in this case.

Reply:
Yeah, I was going to ask about this to...

Reply:
So I checked this on fiddle and it does wrap. That is a bug, as it should be getting placed in a drawer.

Reply:
Logged TINY-6260 to fix the bug in TinyMCE.

Reply:
May of been fixed by: https://github.com/tinymce/tinymce-docs/pull/1612/commits/85c06bb5d09d833d450f339a14642ad85c3ad1db